spec/request/v1/users_items_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe 'User items API request spec', type: :request do
+  let!(:user) { create :user }
+
+  before do
+    3.times { create :item, user: user  }
+  end
+
+  describe '#GET api/user_items' do
+    let(:items_response) { JSON.parse(response.body)['data'] }
+
+    it 'returns unauthorized without a valid api key' do
```

Comment:
this is all fine and good, but what about the actual purpose of this controller? the item stuff?